operator-pipeline-images/operatorcert/entrypoints/reserve_operator_name.py

```diff
@@ -32,6 +32,38 @@ def setup_argparser() -> argparse.ArgumentParser:  # pragma: no cover
     return parser
 
 
+def check_operator_name_registered_for_association(args) -> None:
+    rsp = pyxis.get(
+        urljoin(
+            args.pyxis_url,
+            f"v1/operators/packages?filter=association=={args.association}",
+        )
+    )
+
+    rsp.raise_for_status()
+
+    packages = rsp.json().get("data")
+
+    if packages:
+        # there should only be 1 package for given association/isv_pid
+        package = packages[0]
+        if package["package_name"] != args.operator_name:
+            LOGGER.error(
```

Comment:
These 2 messages will be directly visible to users. Can you reword them a little so the partner understands what's wrong? I mean use wording that is not API specific and is understandable even though the user doesn't know anything about API details.

Reply:
Messages updated